test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1

```diff
@@ -15,7 +15,9 @@ Describe "Get-FormatData" -Tags "CI" {
         $format.TypeNames | Should -HaveCount 2
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'PackageManagement' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```